srcpkgs/lib3mf/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'lib3mf'
+pkgname=lib3mf
+version=2.2.0
+revision=1
+build_style=cmake
+configure_args="-DUSE_INCLUDED_LIBZIP=OFF -DUSE_INCLUDED_ZLIB=OFF
+ -DUSE_INCLUDED_GTEST=OFF -DUSE_INCLUDED_SSL=OFF
+ -DCMAKE_INSTALL_INCLUDEDIR='include/$pkgname' -DCMAKE_INSTALL_PREFIX=/usr
+ $([ "$XBPS_CHECK_PKGS" ] || echo '-DLIB3MF_TESTS=OFF')"
```

Comment:
this would be better as an if block below:

```bash
if [ -z "$XBPS_CHECK_PKGS" ]; then
	configure_args+=" -DLIB3MF_TESTS=OFF"
fi
```
however, I don't think it will affect the final build if the tests are always built

Reply:
The tests have dependencies, so they won't compile correctly by default AFAIK.

Reply:
that can be added to make_depends

Reply:
Not entirely sure what the point of check dependencies are then, but if it's more idiomatic I'll merge them.